doc/docs/usage/installation.md

```diff
@@ -50,6 +58,14 @@ curl -sfL https://github.com/eclipse-ankaios/ankaios/releases/download/<version>
 
 For available versions see the [list of releases](https://github.com/eclipse-ankaios/ankaios/tags).
 
+### Uninstall Ankaios
+
+If Ankaios has been installed with the installation script, it can be uninstalled with:
```

Comment:
Maybe we can give a hint that the initial startup state or the config under /etc/ank is not removed if you want to keep the config. Otherwise if we decide to cleanup the config path as well when uninstalling then no change here is necessary.

Reply:
OK, I will add a sentence telling that the config remains.